esm/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.json",
```

Comment:
Why checking `.mjs` modules using TS?

Reply:
I'm starting to wonder that as well, skipped